pyzscaler/zia/traffic.py

```diff
@@ -387,10 +396,18 @@ def get_closest_diverse_vip_ids(self, ip_address: str):
 
         return recommended_vips
 
-    def list_vpn_credentials(self):
+    def list_vpn_credentials(self, **kwargs):
         """
         Returns the list of all configured VPN credentials.
 
+        Keyword Args:
+            **max_items (int, optional):
+                The maximum number of items to return before stopping iteration.
+            **max_pages (int, optional):
+                The maximum number of pages to request before throwing stopping iteration.
```

Comment:
Quality/Grammar: The maximum number of pages to request before stopping iteration.

Reply:
Thanks! Changeing the other line to "request" as well.